### PR TITLE
Update English RefCard to Pd 0.49.1

### PR DESCRIPTION
**Pull request by NiccoloGranieri**
Updated english refcard to Pd 0.49.1 adding new objects, and doing minor corrections to the object descriptions.

### CODE REVIEW

File: pd-refcard-en.tex [hunk 1]
```diff
@@ -237,8 +245,30 @@
...
+  	pd\~\ stout & run another copy of Pd (for multiprocessing) \\ 
+  	rev1\~\ rev2\~\ rev3\~\ & reverberators \\
+  	bob\~\ & Moog resonant filter model \\
+  	
   \end{tabularx}
+  
+  \medskip{} 
+  \noindent{}
+  \begin{tabularx}{9cm}{X}
+    \tiny{Copyright \copyright{2018} Karim \textsc{Barkati} <\email{karim.barkati@gmail.com}> \& Niccolò \textsc{Granieri} <\email{granieriniccolo@gmail.com}>. Permission is granted to copy, distribute and/or modify this document under the terms of the GNU Free Documentation License, Version 1.3 or any later version published by the Free Software Foundation; with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts.}
```
Comment (kbarkati) [inline pd-refcard-en.tex]: I just would indicate the whole time span of the copyright, in order to let people know when the refcard started:

```
\copyright{2010-2018}
```

File: pd-refcard-it.tex [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  \footnotesize
+
+  \refcardtitle{Modes}
+  \begin{tabularx}{9.1cm}{X}
+    \texttt{ctrl-e} (or \texttt{cmd-e}) toggle between \emph{run} mode (performance) and \emph{edit} mode (programming); this affects how mouse clicks affect the patch.
+  \end{tabularx}
+
+  \refcardtitle{General}
+  \begin{tabularx}{9cm}{>{\tt}l X}
+    bang & output a bang message \\
```
Comment (kbarkati) [inline pd-refcard-it.tex]: An Italian translation is welcome but the source code I can see here is still in English.

**Review by kbarkati — CHANGES_REQUESTED** [review]
Thank you Niccolo for your contribution. It is more complete and still compiles nicely, on a single page.

I have two changes requests before approving your pull request:
1. the copyright time span
2. the Italian translation

File: pd-refcard-en.tex (hunk 1 above)
Comment (NiccoloGranieri) [reply pd-refcard-en.tex]: I'm totally fine regarding the extension of the time span.

File: pd-refcard-it.tex (hunk 2 above)
Comment (NiccoloGranieri) [reply pd-refcard-it.tex]: I didn't do a pull request for the italian translation... I thought I kept that on my private repo until I had time to work on it! Sorry for that.

**Comment by NiccoloGranieri** [comment]
I have never worked with change requests on pull requests. How does it work? Do I need to make the changes on my repo and submit a new pull request?

**Comment by kbarkati** [comment]
> I have never worked with change requests on pull requests. How does it work? Do I need to make the changes on my repo and submit a new pull request?

You just have to push new commits that fix the change requests. No new pull request is needed.

**Comment by NiccoloGranieri** [comment]
All done!

**Comment by NiccoloGranieri** [comment]
@kbarkati any idea how we can update the pdf also on the puredata.info website?

**Comment by kbarkati** [comment]
> @kbarkati any idea how we can update the pdf also on the puredata.info website?

I have just done it: see http://puredata.info/docs/manuals/pdrefcards.
Thank you again for the additions 👍

**Pull request merged**